bagri-server/bagri-server-hazelcast/src/main/java/com/bagri/server/hazelcast/impl/DocumentAccessorImpl.java

```diff
@@ -83,115 +80,8 @@ public DocumentAccessorImpl(SchemaRepository repo, Document doc, long headers, O
 	}
 	
 	@Override
-	public int getFactoryId() {
-		return cli_factory_id;
-	}
-
-	@Override
 	public int getId() {
 		return cli_ServerDocumentAccessor;

```

Comment:
no, this will not work. leave it as it was with inherited from client id

Reply:
If I leave it like this, during deserialisation object of this class is becoming instance if client DocumentAccessorImpl.
Could you please explain what exactly will not work? As all tests are passed..

Reply:
Yep, that is correct, that is what we want: to get an instance of client DocumentAccessrImpl on the client side. It has all data that is required on the client side. If you’ll serialize it with new server Id it’ll be not deserialized on the client side at all, as this server-side id is not known on the client side. This technic of having different classes on the client and server side, but with same serialization I’d is used very often between client and server implementations.